src/Runtime/Runtime.csproj

```diff
@@ -34,4 +34,10 @@
     <Compile Include="..\Shared\SharedAssemblyFileVersion.cs" />
     <Compile Include="..\Shared\SharedAssemblyInfo.cs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
+      <_Parameter1>$(AssemblyNamePrefix)Microsoft.ML.Probabilistic.Compiler,PublicKey=0024000004800000940000000602000000240000525341310004000001000100551f07a755a3e3f2901fa321ab631d13d6192b4e6ac9c87279500f49d6635cde6902587752eff20402f46f6ea9c3d80e827580a799840aaab9a49b1d2597e4c1798ee93c5cb66851e9d22f4d6e8110571f4a2e59f1d760f7be04fb10e7dc43ee7ed2831907731427b9815c5fe7f4888f9933ee7a1ad5d1f293fd8ab834fac1be</_Parameter1>
```

Comment:
Out of curiosity, what is the rationale for having the prefix (different assembly names)? Isn't it sufficient to produce different packages for different partners/customers?

Reply:
I believe the idea is to allow people to consume Infer.NET by submoduling it into their projects -- thus having copies of it with different versioning schemes and so on -- the only way to resolve this seems to be to have something like this. But to be clear: this change isn't adding this mechanism, just fixing it.